### PR TITLE
Add attributes for filesystem device

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

### CODE REVIEW

File: virttest/libvirt_xml/devices/filesystem.py [hunk 1]
```diff
@@ -19,6 +19,43 @@ def __init__(self, type_name='mount', virsh_instance=base.base.virsh):
...
+                                 parent_xpath='/',
+                                 tag_name='alias')
         super(Filesystem, self).__init__(device_tag='filesystem',
                                          type_name=type_name,
                                          virsh_instance=virsh_instance)
+
+    class Binary(base.base.LibvirtXMLBase):
+
+        """
+        Filesystem binary subclass.
```
Comment (chunfuwen) [inline virttest/libvirt_xml/devices/filesystem.py:39]: Better give one example xml here

File: virttest/libvirt_xml/devices/filesystem.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 class Filesystem(base.TypedDeviceBase):
 
-    __slots__ = ('accessmode', 'source', 'target')
+    __slots__ = ('accessmode', 'source', 'target', 'driver', 'binary', 'address', 'alias')
```
Comment (chunfuwen) [inline virttest/libvirt_xml/devices/filesystem.py:13]: I don't find any implementation for address attribute

**Comment by yafu-1** [comment]
Test results:
>>> from virttest.libvirt_xml.devices.filesystem import Filesystem
>>> fs_dev = Filesystem()
>>> fs_dev.accessmode = "passthrough"
>>> fs_dev.driver = {'type': 'virtiofs', 'queue': '1024'}
>>> fs_dev.source = {'dir': '/mnt'}
>>> fs_dev.target = {'dir': 'mount_tag'}
>>> binary_xml = Filesystem.Binary()
>>> binary_xml.path = "/usr/libexec/virtiofsd"
>>> binary_xml.cache_mode = "none"
>>> binary_xml.lock_posix = 'on'
>>> binary_xml.flock = 'on'
>>> fs_dev.binary = binary_xml
>>> print(fs_dev)
<?xml version='1.0' encoding='UTF-8'?>
<filesystem accessmode="passthrough" type="mount"><driver queue="1024" type="virtiofs" /><source dir="/mnt" /><target dir="mount_tag" /><binary path="/usr/libexec/virtiofsd"><cache mode="none" /><lock flock="on" posix="on" /></binary></filesystem>

File: virttest/libvirt_xml/devices/filesystem.py (hunk 1 above)
Comment (yafu-1) [reply virttest/libvirt_xml/devices/filesystem.py]: Updated.

File: virttest/libvirt_xml/devices/filesystem.py (hunk 2 above)
Comment (yafu-1) [reply virttest/libvirt_xml/devices/filesystem.py]: Updated.

**Pull request merged**